src/Sylius/Bundle/UserBundle/Controller/SecurityController.php

```diff
@@ -56,12 +59,13 @@ public function logoutAction(Request $request)
      * Renders the login template with the given parameters. Overwrite this function in
      * an extended controller to provide additional data for the login template.
      *
-     * @param array $data
+     * @param string $view The view name
+     * @param array  $data
      *
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    protected function renderLogin(array $data)
+    protected function renderLogin($view, array $data)
```

Comment:
As I answered eariler (https://github.com/Sylius/Sylius/pull/3193#discussion_r38394962), Symfony's render method has a `view` variable, but for me also better name is `template`. Renaming then.